main.py

```diff
@@ -185,7 +185,7 @@
         if int(scoutData['matchNum']) == matchNum:
             scoutIndex = scoutNames.index(scoutData['scoutName'])
             scoutGamePieceCounts[scoutData['scoutName']] += scoutData['autoCoralL1'] + scoutData['autoCoralL2'] + scoutData['autoCoralL3'] + scoutData['autoCoralL4'] + scoutData['teleopCoralL1'] + scoutData['teleopCoralL2'] + scoutData['teleopCoralL3'] + scoutData['teleopCoralL4'] + scoutData['teleopProcessorAlgae'] + scoutData['autoProcessorAlgae']
-            if int(scoutData['teamNum']) in redAllianceTeamNums and (red_correction is None or int(scoutData['teamNum']) != int(red_correction['teamNum'])):
+            if int(scoutData['teamNum']) in redAllianceTeamNums and (red_corrections[int(correction['teamNum'])] is None or int(scoutData['teamNum']) != int(red_corrections[int(correction['teamNum'])]['teamNum'])):
```

Comment:
I think this should be `red_corrections[int(scoutData['teamNum'])] is None`, without an or statement

Reply:
(Same for blue)